packages/pn-personafisica-webapp/src/pages/Contacts.page.tsx

```diff
@@ -97,6 +97,7 @@ const Contacts = () => {
             title={t('title')}
             subTitle={subtitle}
             variantSubTitle={'body1'}
+            ariaLabel="Recapiti"
```

Comment:
Use the translation hook, same as `title`.

Reply:
done